### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by TeemSucks**
adds 'start' command and adds an MD file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+To download and install a bot run these following commands in command prompt:
+```sh
+git clone LINK
+cd a-bot
+```
+
+You also have to install node, to do so run this command:
+```sh
+npm i
```
Comment (AllMesi) [inline README.md:13]: this isn't installing node tho, this is just installing the dependencies

install node here https://nodejs.org/en

**Review by AllMesi — APPROVED** [review]
good for now

**Review by AllMesi — APPROVED** [review]
good

File: README.md (hunk 1 above)
Comment (TeemSucks) [reply README.md:13]: fixed i think

**Pull request merged**